index.html

```diff
@@ -35,39 +35,43 @@ <h1 class="welcome">Welcome</h1>
 					</div>
 				</div>
 
-			</div>
+			</nav>
 
 			<!-- intro -->
-			<div class="intro">
-				<div class="primary">
-					<p class="primary-text"
-					   id="primary">Hey there. I’m Jared 
-					   <span id="career">I’m a software
-						   developer</span></p>	
+			<header class="intro">
+				<div>
+					<h1 class="primary-text"
+					    id="primary">Hey there. I’m Jared 
+				</div>
+				<div>
+					<h1 class="primary-text"id="career">I’m a software
+						developer</h1></h1>	
 				</div>
-			</div>
+			</header>
 
 			<!-- Support text -->
-			<div class="support-text">
-				<p id="support-text"> I can help you build
-				a product , feature or website Look
-				through some of my work and experience! If
-				you like what you see and have a project
-				you need coded, 
-				don’t hestiate to contact me.</p>
-			</div>
+			<header class="support-text">
+				<h1 id="support-text"> I can help you build
+					a product , feature or website Look
+					through some of my work and experience! If
+					you like what you see and have a project
+					you need coded, 
+					don’t hestiate to contact me.</h1>
+			</header>
+
+			<nav class="social">
```

Comment:
- [ ] I think you didn't make the background the positioned the same as the figma design so kindly add it to make your design identical to the Figma design as much as possible

- Your design
![image](https://user-images.githubusercontent.com/58553711/198180001-4b7575bd-81a8-43bb-b068-35470e40af0b.png)


- Figma
![image](https://user-images.githubusercontent.com/58553711/198179947-244c89db-c914-4aab-9cbd-7d0e9d38f09c.png)